src/core/dynamic_batch_scheduler.cc

```diff
@@ -411,20 +355,20 @@ DynamicBatchScheduler::SchedulerThread(
       // If no requests are to be handled, wait for notification or
       // for the specified timeout before checking the queue again.
       if (wait_microseconds > 0) {
-        idle_scheduler_thread_cnt_++;
         std::chrono::microseconds wait_timeout(wait_microseconds);
         cv_.wait_for(lock, wait_timeout);
-        idle_scheduler_thread_cnt_--;
       }
     }
 
-    if (wake_thread) {
-      cv_.notify_one();
-    }
-
     if (!requests.empty()) {
-      OnSchedule_(runner_id, std::move(requests));
+      model->Instances()[instance_index]->Schedule(std::move(requests));
```

Comment:
Just want to make sure that later on the thread pool will be invoked in `Schedule` either to delegate the execution to other thread or to let the scheduler thread to call into the instance execution?

Reply:
Yes.. For multiple instances, the requests will be delegated to BackendThread within TritonModelInstance